### DISCUSSION TITLE
[Discussion] Merging Chrome tabs

### DISCUSSION THREAD

**Discussion by mike1808** [nikitabobko/AeroSpace#632] [general]
I wanted to discuss with users of Aerospace how they merge Chrome tabs together. The usual way to do it is to drag the tab and drop it in the window where you want the tab to go. However, with Aerospace as soon you deattach the tab from the window Chrome makes a separate window and Aersopace tries to align it.  I wanted to know what can be done to make merging Chrome tabs easier with Aerospace. Possibly ignore the window that being dragged? 

P.S. There are no problems with Firefox because it doesn't create a window when trying to move the tab.

**Comment by jamsajones** [comment]
I use this Chrome extension. https://chromewebstore.google.com/detail/move-tabs-between-windows/dcgkknngicjdagcjljfbhgdimlfhafag. Better than drag and drop any day.

**Comment by nikitabobko** [comment]
And I can also recommend https://chromewebstore.google.com/detail/open-as-popup/ncppfjladdkdaemaghochfikpmghbcpc to extract the current tab as a popup

When I wait for something to complete (CI to complete, Slack message to arrive, Delivery page with a progress bar), I tend to extract the tab and put it as a standalone window to the external monitor as a dashboard

**Comment by paulodiovani** [comment]
My suggestion is that Aerospace don't try to resize windows and make room for a new window while this one is dragged, but only when dropped.

Meanwhile, an option is to make the second window float before trying to join it.
